templates/common/_base/units/machine-config-daemon-firstboot.service.yaml

```diff
@@ -8,8 +8,7 @@ contents: |
   # Removal of this file signals firstboot completion
   ConditionPathExists=/etc/ignition-machine-config-encapsulated.json
   After=machine-config-daemon-pull.service
-  Before=crio.service crio-wipe.service
```

Comment:
out of curiosity why are you removing this dependency? the firstboot service does have scenarios today where it would make a call to podman

Reply:
crio-wipe should run before firstboot, which is ensured by the After= field in mcd-pull. Should I add an After=crio-wipe.service here too?

Reply:
Ah right I forgot that explicit dependency. No that's fine as is